iris-mpc-store/src/lib.rs

```diff
@@ -179,14 +179,23 @@ impl Store {
             let start_id = 1 + partition_size * i;
             let end_id = start_id + partition_size - 1;
 
-            let partition_stream = sqlx::query_as::<_, StoredIris>(
-                "SELECT * FROM irises WHERE id BETWEEN $1 AND $2 AND last_modified_at >= $3",
-            )
-            .bind(start_id as i64)
-            .bind(end_id as i64)
-            .bind(min_last_modified_at)
-            .fetch(&self.pool)
-            .map_err(Into::into);
+            let partition_stream = match min_last_modified_at {
+                Some(min_last_modified_at) => sqlx::query_as::<_, StoredIris>(
+                    "SELECT * FROM irises WHERE id BETWEEN $1 AND $2 AND last_modified_at >= $3",
+                )
+                .bind(start_id as i64)
+                .bind(end_id as i64)
+                .bind(min_last_modified_at)
+                .fetch(&self.pool)
+                .map_err(Into::into),
+                None => sqlx::query_as::<_, StoredIris>(
+                    "SELECT * FROM irises WHERE id BETWEEN $1 AND $2",
+                )
+                .bind(start_id as i64)
+                .bind(end_id as i64)
+                .fetch(&self.pool)
+                .map_err(Into::into),
+            };
```

Comment:
cargo fmt removes the curl brackets around Some and None which makes it less readable :/